### PR TITLE
[REFACTOR]: Identity List + Detail & misc. fixes

### PR DESCRIPTION
**Pull request by okaymak**

### CODE REVIEW

File: .idea/dictionaries/tiqr.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<component name="ProjectDictionaryState">
```
Comment (dzolnai) [inline .idea/dictionaries/tiqr.xml:1]: This file is in .gitignore, but still committed. Is this deliberate?

File: app/src/main/java/org/tiqr/authenticator/util/databinding/BindingAdapters.kt [hunk 2]
```diff
@@ -90,4 +114,43 @@ fun TextView.appName(appName: String) {
 fun View.openBrowser(url: String) {
     if (url.isEmpty()) return
     setOnClickListener(Navigation.createNavigateOnClickListener(MainNavDirections.openBrowser(url)))
+}
+
+/**
+ * Add dividers
+ */
+@BindingAdapter(value = ["dividers"])
```
Comment (dzolnai) [inline app/src/main/java/org/tiqr/authenticator/util/databinding/BindingAdapters.kt:122]: These can be simplified to:

    @BindingAdapter("dividers")

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -82,4 +84,17 @@
...
 
+    <!-- Identity -->
+    <string name="identity_title">Identités</string>
+    <string name="identity_blocked">Bloqué</string>
+    <string name="identity_blocked_message">Attention: ce compte est bloqué et doit être re-activé.</string>
+    <string name="identity_delete_title">Êtes-vous sûr?</string>
+    <string name="identity_delete_message">Ce compte ne peut plus être utilisé lorsqu\'il est supprimé.</string>
+    <string name="identity_biometric_usage">Utiliser la biométrie</string>
+    <string name="identity_biometric_upgrade">Mise à niveau vers l\'utilisation de la biométrie</string>
+
```
Comment (dzolnai) [inline app/src/main/res/values-fr/strings.xml:95]: Lots of empty lines

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package org.tiqr.authenticator.buildsrc
 
-private const val kotlinVersion = "1.3.71"
+private const val kotlinVersion = "1.3.72"
+private const val navigationVersion = "2.3.0-alpha05"
```
Comment (dzolnai) [inline buildSrc/src/main/kotlin/Dependencies.kt:4]: Do you need to use alpha versions?

File: .idea/dictionaries/tiqr.xml (hunk 1 above)
Comment (okaymak) [reply .idea/dictionaries/tiqr.xml:1]: Yes. The folder `.idea` is already ignored, but `.idea/dictionaries` is whitelisted, so this file gets added.

File: app/src/main/java/org/tiqr/authenticator/util/databinding/BindingAdapters.kt (hunk 2 above)
Comment (okaymak) [reply app/src/main/java/org/tiqr/authenticator/util/databinding/BindingAdapters.kt:122]: I like the label :blush:

File: app/src/main/res/values-fr/strings.xml (hunk 3 above)
Comment (okaymak) [reply app/src/main/res/values-fr/strings.xml:95]: There will be more coming soon, filling up the empty lines :blush:

File: buildSrc/src/main/kotlin/Dependencies.kt (hunk 4 above)
Comment (okaymak) [reply buildSrc/src/main/kotlin/Dependencies.kt:4]: Not yet, but gonna need it to send result back to a fragment (which is only supported from version 2.3.0).

File: .idea/dictionaries/tiqr.xml (hunk 1 above)
Comment (dzolnai) [reply .idea/dictionaries/tiqr.xml:1]: Indeed, I misread the rule :)

**Pull request merged**